### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by bleachjade**
- add google auth, fix profile page style, add arrange date & time, add more detail in sign up page.

### CODE REVIEW

**Comment by forfeen** [comment]
It's nicely to merge.

File: Kvent/forms.py [hunk 1]
```diff
@@ -37,11 +41,12 @@ class SignUpForm(UserCreationForm):
...
-        fields = UserCreationForm.Meta.fields + ( 'email','username', 'password1')
+        fields = UserCreationForm.Meta.fields + ( 'email','username','first_name','last_name','phone_num','address','password1' )
 
         widgets = {
             'email': forms.EmailInput(attrs={'class': 'form-control', 'placeholder': 'Email'}),
-            'username': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Username'}),
-            'password1': forms.PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Password'}),
-            'password2': forms.PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Password Again'}),
+            # 'username': forms.TextInput(attrs={'class': 'form-control', 'placeholder': 'Username'}),
+            # 'password1': forms.PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Password'}),
```
Comment (forfeen) [inline Kvent/forms.py]: Delete this

File: mysite/settings.py [hunk 2]
```diff
@@ -54,8 +56,11 @@
 
 AUTHENTICATION_BACKENDS = (
      # username/password authentication
-    'django.contrib.auth.backends.ModelBackend',  
- )
+    'social_core.backends.google.GoogleOAuth2',
+    # 'social_auth.backends.google.GoogleBackend',
```
Comment (forfeen) [inline mysite/settings.py]: Delete this

**Comment by BenZacs** [comment]
I think everythimg is ok for merge.

**Pull request merged**